### PR TITLE
Bug 1934152: using Tang is supported on vSphere

### PR DESCRIPTION
**Pull request by miabbott**

### CODE REVIEW

**Comment by miabbott** [comment]
cc: @jlebon @bobfuru

**Review by bgilbert — APPROVED** [review]
LGTM

**Review by jlebon — APPROVED** [review]

File: modules/installation-special-config-storage.adoc [hunk 1]
```diff
@@ -22,7 +22,12 @@ This feature:
 There are two different supported encryption modes:
 
 * TPM v2: This is the preferred mode. TPM v2 stores passphrases in a secure cryptoprocessor.
-* Tang: To use Tang to encrypt your cluster, you need to use a Tang server. Clevis implements decryption on the client side. Tang encryption mode is only supported for bare metal installs.
+* Tang: To use Tang to encrypt your cluster, you need to use a Tang server. Clevis implements decryption on the client side.
+
+[IMPORTANT]
+====
+The use of Tang encryption mode to encrypt your disks is only supported for bare metal and vSphere UPI installs.
```
Comment (bobfuru) [inline modules/installation-special-config-storage.adoc:29]:
```suggestion
The use of Tang encryption mode to encrypt your disks is only supported for bare metal and vSphere installations on user-provisioned infrastructure.
```
(We no longer refer to UPI/IPI and thus need to spell it out here. Thanks!)

**Review by bobfuru — COMMENTED** [review]
One minor request but otherwise this LGTM. This should apply to 4.6+ if I'm not mistaken.

Let me know when it's ready for a merge. Thanks!

**Comment by miabbott** [comment]
Squashed commits + ready to merge!

**Pull request merged**